### ISSUE TITLE
导入time-series问题

### ISSUE THREAD

**Issue by shangguanyifeng** [PaddlePaddle/Paddle#12361]
x_ = fluid.layers.data(name='x_', shape=[-1,seq_size,input_dim], dtype='float32', lod_level = 1)
y_ = fluid.layers.data(name='y_', shape=[-1,seq_size], dtype='float32', lod_level = 1)
def seq_reader_creator(seq_path):
    def reader():
        with open(seq_path, 'rb') as f:
            data = pickle.load(f)
            normalized_data = (data - np.mean(data)) / np.std(data)
        for i in xrange(len(normalized_data) - seq_size - 1):
            yield np.expand_dims(normalized_data[i : i + seq_size], axis=1).tolist(), normalized_data[i + 1 : i + seq_size + 1].tolist()  # a single entry of data is created each time
    return reader

train_reader_raw = seq_reader_creator("../bandwidth_dataset/data_interval_300.pickle")

train_reader = paddle.batch(
train_reader_raw,
batch_size=batch_size)
feeder = fluid.DataFeeder(place=place, feed_list=[x_, y_])

loss = exe.run(
            fluid.default_main_program(),
            feed=feeder.feed(data),
            fetch_list=[avg_cost])

**Comment by Superjomn** [comment]
是啥问题？

另外，代码可以用

https://help.github.com/articles/creating-and-highlighting-code-blocks/

格式化下，不然很难看出来

**Comment by lucywsq** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，
我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持!
